app/javascript/dashboard/components/widgets/conversation/Message.vue

```diff
@@ -269,6 +274,8 @@ export default {
         copy: this.hasText,
         delete: this.hasText || this.hasAttachments,
         cannedResponse: this.isOutgoing && this.hasText,
+        replyTo: false,
+        // replyTo: !this.data.private && this.inboxSupportsReplyTo,
```

Comment:
Let us use feature flag here. Otherwise looks fine.

Reply:
Noted, any reference to how I can add that. Couldn't find any specific utility except for `featureFlags.js` for holding the keys

Reply:
Should I be adding it to `features.yml`?

Reply:
Let us pair.